src/main/java/io/vertx/codegen/type/ClassKind.java

```diff
@@ -85,9 +81,7 @@ public static ClassKind getKind(
       String fqcn,
       boolean isDataObjectAnnotated,
```

Comment:
This is not required anymore, you can remove it

Reply:
right, thanks for spotting it.